src/components/InputPillGroup/InputWithButtonPillGroup.controlled.tsx

```diff
@@ -13,13 +14,42 @@ export const ControlledInputWithButtonPillGroup = <
   setValue,
   totalAmount,
   totalAmountDecimals,
+  feesAndLimits,
```

Comment:
I think it would be better to pass here just the validate function. `InputWithButtonPills` is a generic component used in more places in the app, you can define this function in `SendAmount`, it's used only in Aggregator